float32 goal_position_x
float32 goal_position_y
float32 goal_position_z
---
string status
---
bool success